config/mainnet.dhall

```diff
@@ -0,0 +1,5 @@
+{ keyType      = ""
```

Comment:
This file should accept OS settings as input and then you can just do:

`configurationKey = mainnet_wallet_${os.updateSystemName}"

Reply:
But we already have an OS-specific file, why would we want another one?